frontend/13f-app/src/components/companydata.js

```diff
@@ -0,0 +1,25 @@
+import React, { useState, useEffect } from 'react'
+import { useLocation } from 'react-router-dom'
+import axios from 'axios'
+
+function Company () {
+  const [company, setCompany] = useState()
+  const location = useLocation()
+  const param = new URLSearchParams(location.search)
+  const name = param.get('name')
```

Comment:
You're going to want to use [`useParams`](https://reactgo.com/react-get-url-params/) here instead